api/src/services/__tests__/meetingsService.ts

```diff
@@ -152,6 +152,11 @@ describe('meetingsService', () => {
         [agendaOwningParticipantId, 6, noteText, sortOrder],
         [meetingNoteId]
       );
+      mockQuery(
+        'select `id`, `agenda_owning_participant_id`, `authoring_participant_id`, `sort_order`, `note_text`, `created_at` from `meeting_notes` where `id` = ?',
+        [meetingNoteId],
+        [{ id: 5 }]
```

Comment:
The mocked SQL and this response don't match. The query selects six fields, but only `id` is in the object in the response.

Reply:
Okay, you're right. I added additional variables to represent the `meetingNote` object which is also the response of that query. Then I updated the expected response for the test to also equal `meetingNote`.